app/assets/javascripts/misc/i18n-mode.js

```diff
@@ -5,15 +5,14 @@ document.addEventListener('DOMContentLoaded', function() {
   const inputs = document.querySelectorAll('input[type="submit"]');
 
   if (inputs) {
-    for (var i = 0; i < inputs.length; i++) {
-      const input = inputs[i];
+    [].slice.call(inputs).forEach((input) => {
```

Comment:
The `i` is still being used in `pw-toggle.js` to generate a unique ID for each template the `togglePW` function generates.

We don't need the `i` in this case because the function passed to `forEach` doesn't care about the specific iteration.